.gitignore

```diff
@@ -8,8 +8,4 @@
 *.iml
 
 # bin
-/target/
-
-# xjc generation
-src/main/java/org/lsc/configuration/
-lsc.episode
\ No newline at end of file
+/target/
```

Comment:
You may prefer I squash all the commit in one ?
(Sorry, I'm not totally comfortable with pull requests yet)